### DISCUSSION TITLE
请问在情景模式中如何打开 SchemeURL？

### DISCUSSION THREAD

**Discussion by ListenV** [Tornaco/Thanox#601] [Q&A] [answered]
## 我的目的

我想快速打开小程序的健康码

## 实现方式

根据我所查到的资料，使用 `SchemeURL` 打开健康码是一种方式

`SchemeURL` 类似于这种： `weixin://app/wxd930ea5d5a258f4f/jumpWxa/?userName=gh_21c954017d6d`

我的思路是：先创建一个快捷方式，快捷方式打开的时候，触发 action **打开SchemeURL**，跳到小程序健康码

## 问题

我查了 Thanox 文档，没有 打开SchemeURL 相关说明

请问该如何实现？

**Comment by ListenV** [comment]
大家都不用discussions吗？

**Comment by Tornaco** [comment, marked as answer]
Android页面跳转直接用Android自带API就可以了，情景模式开放了Android Context，可以用它打开对应页面：

```java
context.startActivity(new android.content.Intent(android.content.Intent.ACTION_VIEW, android.net.Uri.parse("https://www.bing.com")).addFlags(android.content.Intent.FLAG_ACTIVITY_NEW_TASK))
```

上面的代码可以用在情景模式中，但是注意引号要加反斜杠`\`转义

**Reply by ListenV** [reply]
好的，太感谢了
